webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala

```diff
@@ -75,7 +77,14 @@ object OntologiesRouteV2 extends Authenticator {
                             InputValidation.toIri(internalOntologyIri, () => throw BadRequestException(s"Invalid named graph Iri: '$internalOntologyIri'"))
                     }.toSet
 
-                    val requestMessage = NamedGraphEntitiesGetRequestV2(internalOntologyIris, userProfile = userProfile)
+                    val params: Map[String, String] = requestContext.request.uri.query().toMap
+                    val allLanguages = InputValidation.optionStringToBoolean(params.get(ALL_LANGUAGES))
```

Comment:
Error handling is needed here in case the user submits a value for `allLanguages` that cannot be converted by `toBoolean`.

Reply:
`java.lang.IllegalArgumentException` is thrown if String cannot be converted to Boolean.